src/common/problem_definition.cc

```diff
@@ -1,151 +1,155 @@
-#include <deal.II/base/utilities.h>
+#include "problem_definition.h"
 
 #include <sstream>
 #include <utility>
 #include <iomanip>
 
-#include "problem_definition.h"
-
-ProblemDefinition::ProblemDefinition () {}
+#include <deal.II/base/utilities.h>
 
-ProblemDefinition::~ProblemDefinition() {}
+dealii::ParameterHandler bparams::GlobPrm;
```

Comment:
We should probably put this in a function like this:
```
dealii::ParameterHandler& bparams::GlobalParam() {
    static dealii::ParameterHandler global_prm;
    return global_prm;
}
```
And then just use `bparams::GlobalParam()` to access.